scripts/sagas/poll.js

```diff
@@ -0,0 +1,51 @@
+import { push as updatePath } from "react-router-redux";
+import { put } from "redux-saga/effects";
+import btoa from "btoa";
+import KintoClient from "kinto-http";
+
+import { voteSubmitted, pollCreated } from "../actions/poll";
+
+
+function* createBucket(client, bucket) {
+  try {
+    // Create the bucket in case it doesn't exist.
+    const permissions = {"collection:create": ["system.Everyone"]};
+    const headers = {Authorization: "Basic " + btoa(`token:${Math.random()}`)};
+    yield client.createBucket(bucket, {headers, permissions, safe: true});
+  } catch(e) {
+    // Ignore error if it already exists.
+    if (!/HTTP 4??/.test(e.message)) {
```

Comment:
What does this do? Never seen the double question mark. Also I would expect we distinguish between 400 and 404 for instance.